web_src/js/bootstrap.js

```diff
@@ -29,17 +29,26 @@ export function showGlobalErrorMessage(msg) {
  * @param {ErrorEvent} e
  */
 function processWindowErrorEvent(e) {
+  const err = e.error ?? e.reason;
+  const assetBaseUrl = String(new URL(__webpack_public_path__, window.location.origin));
+
+  // error is likely from browser extension or inline script. Do not show these in production builds.
+  if (!err.stack?.includes(assetBaseUrl) && (window.config?.runModeIsProd ?? true)) return;
```

Comment:
`?? true`?
Why?
If we are not sure, show the user errors

Reply:
I defaulted it to hide errors when `window.config` is not there for whatever reason. It fits the style of this file I guess where the other check for the variable is also with `?.` and there is a specific error message related to `window.config` being absent so we don't want to run into this error before that error message was emitted.

Reply:
Ah I see what you mean now. I guess we can default to showing it.